requests.go

```diff
@@ -148,3 +155,15 @@ func (c *Client) put(path string, stream io.Reader) int {
 
 	return rs.StatusCode
 }
+
+func (c *Client) createParentCollection(itemPath string) (err error) {
+
+	parentPath := path.Dir(itemPath)
+
+	err = c.MkdirAll(parentPath, 0755)
```

Comment:
no need for wrap, just return the result.

Reply:
done in [0a18eef](https://github.com/studio-b12/gowebdav/pull/22/commits/0a18eef38649e62b68935bc2e6a9de17391124dd)